src/platform-apps/channels/util/api.ts

```diff
@@ -0,0 +1,15 @@
+import { searchMentionableUsers } from '../../../store/channels/api';
+import { User } from '../../../store/channels';
+
+export async function searchMentionableUsersForChannel(
+  channelId: string,
+  search: string,
+  // When the api is updated to only return channel users we can remove this argument
+  validUsers: Array<User>,
+  apiSearch = searchMentionableUsers
+) {
```

Comment:
let's add return type

Reply:
I prefer to let typescript infer the return type generally. Is there a particular reason to strictly define it?